### ISSUE TITLE
Accessibility: Make ZeroClipboard work for Enter key presses

### ISSUE THREAD

**Issue by JamesMGreene** [zeroclipboard/zeroclipboard#80]
Currently, ZeroClipboard attaches itself to glued elements on `mouseover` and injects into the user's clipboard on `click`.  This is awesome.

However, everything I've read about the [Flash clipboard API](http://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/desktop/Clipboard.html) suggests that we can also trigger the injection on keyboard events, e.g. the user pressing the Enter key.  Not necessarily a high priority feature but it is something I'd like us to _attempt_ to get working at some point in the future....

Note: We may not succeed. :confused: 

**Requirements:**
1. Glued HTML elements now need a "focus" handler, which would then [focus on the Flash object](http://stackoverflow.com/a/8727763/471696) instead.
   - Here's [another focusing approach, using SWFObject.js and jQuery](https://github.com/englandrp/Cross-browser-Flash-tabbing-and-focus-solution) that claims to work cross-browser as well.
   - Here's a ["SWFFocus" technique](http://www.w3.org/TR/WCAG20-TECHS/FLASH17) described by the W3C accessibility group... should maybe _start_ with that.
2. The ActionScript would need a new [`KEY_DOWN` event handler listening for the Enter key](http://stackoverflow.com/a/2297738/471696).

**Possible points of failure:**
1. Some sources speculate that this is only possible if we change `wmode` to `opaque`. I _believe_ doing so would make the ZC object a visible layer, which would be unacceptable.
2. Stealing focus to the Flash object will probably cause issues if someone was trying to tab/arrow _past_ the element that ZC is glued to (rather than trying to tab/arrow _to_ it to use ZC). Possible workarounds:
   - Again, first refer to the ["SWFFocus" technique](http://www.w3.org/TR/WCAG20-TECHS/FLASH17) described by the W3C accessibility group.
   - This may be possible to avoid if the ActionScript `KEY_DOWN` event handler we add captures tab/arrow and defers it back to JS somehow (including focusing back on the element it came from last).
   - May be able to [add a JS event handler to the Flash object/embed element that swaps it in/out of the z-index/layer when needed](http://stackoverflow.com/a/13486194/471696)

**Comment by kevgrig** [comment]
This would be a nice feature

**Comment by cjolif** [comment]
I thought github was using ZeroClipboard, and Github clone URL copy is working not only on click but Enter as well. Does it mean something has been done here despite the issue still being open? Or is that a Github "only" feature?

**Comment by cjolif** [comment]
Ok, looks like this is working on Chrome only.
